docs/docs/basics/03-accounts.md

```diff
@@ -165,3 +165,127 @@ https://github.com/cosmos/cosmos-sdk/blob/v0.46.0/proto/cosmos/tx/v1beta1/tx.pro
 * `ed25519`
 
 * `ExportPrivKeyArmor(uid, encryptPassphrase string) (armor string, err error)` exports a private key in ASCII-armored encrypted format using the given passphrase. You can then either import the private key again into the keyring using the `ImportPrivKey(uid, armor, passphrase string)` function or decrypt it into a raw private key using the `UnarmorDecryptPrivKey(armorStr string, passphrase string)` function.
+
+### Create New Key Type
+
+To create a new key type for using in keyring, `keyring.SignatureAlgo` interface must be fulfilled.
+
+```go
+// cosmos-sdk/crypto/keyring/signing_algorithms.go
```

Comment:
We have a plugin that allows to display codeblock from GitHub.
Could you use that and using the `v0.47.0-rc1` branch.

Reply:
https://github.com/cosmos/cosmos-sdk/blob/main/docs/DOC_WRITING_GUIDELINES.md#code-snippets

Reply:
done :)